compiler/rustc_hir_typeck/src/intrinsicck.rs

```diff
@@ -46,7 +46,10 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         let from = normalize(from);
         let to = normalize(to);
         trace!(?from, ?to);
-
+        if from.has_non_region_infer() || to.has_non_region_infer() {
+            // We can't check anything if there are inference variables.
```

Comment:
use `tcx.sess.delay_span_bug` here, please

Reply:
Do I need to generate different messages for `form` and `to`? If so, how do I get their corresponding span here? Thanks!

Reply:
no, it's a delayed bug so it should never be printed -- it just needs to explain what it's for.

Reply:
Got it. Thanks!